denops/@ddu-kinds/git_branch.ts

```diff
@@ -132,4 +136,29 @@ export class Kind extends BaseKind<Params> {
   params(): Params {
     return {};
   }
+
+  async getPreviewer({
+    item,
+  }: GetPreviewerArguments): Promise<Previewer | undefined> {
+    const { cwd, refName } = item.action as ActionData;
+    return await Promise.resolve({
+      kind: "terminal",
+      cmds: [
+        "sh",
+        "-c",
+        `${
+          [
+            "cd",
+            cwd,
+            "&&",
+            "git",
+            "log",
+            refName.remote == ""
+              ? refName.branch
+              : `${refName.remote}/${refName.branch}`,
+          ].join(" ")
+        }`,
+      ],
```

Comment:
```suggestion
      cmds: [
        "git",
        "-C",
        cwd,
        "log",
        refName.remote == ""
          ? refName.branch
          : `${refName.remote}/${refName.branch}`,
      ],
```


It may be solve by `-C` option for `git` command